engines/libaio.c

```diff
@@ -79,6 +118,23 @@ static int fio_libaio_prep(struct thread_data fio_unused *td, struct io_u *io_u)
 	return 0;
 }
 
+static void fio_libaio_prio_prep(struct thread_data *td, struct io_u *io_u)
+{
+	struct libaio_options *o = td->eo;
+	struct libaio_data *ld = td->io_ops_data;
+	if (rand_between(&ld->prio_state, 0, 99) < o->prio_percent) {
+		dprint(FD_IO, "Enable PRIO \n");
```

Comment:
Will these `dprint` statements have to be evaluated on every I/O?

Reply:
Yes. Since the decision is made on a per-I/O basis, I'm not sure if there's a less verbose way to include this debug information. Do you think this will produce too much output and should be taken out?